api.planx.uk/modules/file/routes.ts

```diff
@@ -36,16 +37,23 @@ router.post(
 
 router.get(
   "/file/public/:fileKey/:fileName",
-  validate(downloadFileSchema),
+  validate(hostedFileSchema),
   publicDownloadController,
 );
 
 router.get(
   "/file/private/:fileKey/:fileName",
   useNoCache,
   useFilePermission,
-  validate(downloadFileSchema),
+  validate(hostedFileSchema),
   privateDownloadController,
 );
 
+router.delete(
```

Comment:
would definitely like to hear whether anyone thinks there's a risk associated with exposing this endpoint? I've made it accessible only to anyone with platform level admin powers.

the utility of it here is that we can upload as many files to staging S3 as we like during load testing without worrying about incurring additional cost, because we can also delete them immediately

Reply:
Exposing to `platformAdmins` seems fine by me ! 

"Public" files are pretty exclusively flow content (eg vector drawings like roof pitch examples) used throughout services uploaded by editors; all have original copies on OSL's Google Drive, so low risk here! (Whereas "/file/private/..." are uploaded by applicants, so it's ideal to keep deletion of those restricted to session sanitation jobs.)